src/Microsoft.Performance.Toolkit.Engine.Tests/ToolkitEngineTests.cs

```diff
@@ -534,6 +534,16 @@ public void BuildTable_SingleRowTable()
             }
         }
 
+        //[TestMethod]
+        //[FunctionalTest]
```

Comment:
Do we still need this?

Reply:
Thanks, I'll take a look. The problem with working on this on/off for 1.5 months...

Reply:
A couple tests for internal tables have been added now.